common/lib/capa/capa/safe_exec/__init__.py

```diff
@@ -1,3 +1,4 @@
 """Capa's specialized use of codejail.safe_exec."""
 
 from .safe_exec import safe_exec, update_hash
+
```

Comment:
Make the linter fail until codejail versioning is sorted out, so I don't accidentally merge.